css/views/settings-payment-method.css

```diff
@@ -0,0 +1,10 @@
+.payment-method-description {
+	background: #f7f7f7;
+	font-size: 1rem;
+	margin-bottom: 1rem;
+	margin-bottom: 1rem;
+	padding: 1rem;
+}
+.form-description {
```

Comment:
Should go in [css/forms.css](https://github.com/samotari/crypto-terminal/blob/master/css/forms.css)